src/audio/src/src/src_ipc4_int32_10_21_2500_5000.h

```diff
@@ -5,7 +5,7 @@
  */
 
 /** \cond GENERATED_BY_TOOLS_TUNE_SRC */
-#include <sof/audio/src/src.h>
+#include "../src.h"
```

Comment:
Why is the location changed? The coefficients are generic polyphase FIR coefficients for a fractional ratio and could be in theory used by some other SRC like component. It's not a big deal to move and move back if need but was this causing issues?

If needed please make this change also to SRC coefficients generator. It can be a separate PR. These files must be directly generated without any manual edits done.

Reply:
@singalsu , intention is move all related header to one folder, you can check volume part, all volume related moved to src/audio/volume folder.

Reply:
OK!

Can you please make another PR to change this similarly https://github.com/thesofproject/sof/blob/2c6c1b4d4fb636ea78dfe1136737c4b6baf5b21c/tools/tune/src/src_export_coef.m#L53

Try to run e.g. https://github.com/thesofproject/sof/blob/main/tools/tune/src/src_ipc4_int32.m in octave

and check that you get similar h-files as these edited.

Reply:
Also I think directory name "coefficients" would be better than "src" for the moved header files.